### PR TITLE
Update funding addresses for Deuteronomy upgrade

### PR DESCRIPTION
**Pull request by schancel**
As per title

### CODE REVIEW

**Comment by maff1989** [comment]
@ajugorji any update on this? do you need some help getting the changes committed?

**Comment by maff1989** [comment]
@schancel FYI the address list updates are now complete

**Review by maff1989 — APPROVED** [review]

File: src/consensus/addresses.h [hunk 1]
```diff
@@ -110,20 +110,28 @@ Consensus::CoinbaseAddresses AddressSets = {
...
-            "lotus_16PSJQhqxYBFLYmqkZqvs8cf6ydaaXRxJBCyHCffb",
+            "lotus_16PSJHaDBFwmD1HPVWv3zLo3dbXMZTnCgBRzqFusu",
+            "lotus_16PSJMKqnLMJYonvMdn4mVqMb3bG5C76spwKuW4DL",
+            "lotus_16PSJHedfWvFixJzv5pV7g3FY2Z62kT9NprSqfYxW",
+            "lotus_16PSJJeYibqH7K189RbonEcoR7djvw54HWBebPQLY",
+            "lotus_16PSJJPzQYbd5JCX8UpukirFGz9hmCWFjGAhByDip",
+            "lotus_16PSJLr6AKaWTgioeJbjUWFAdq7yVDG17PQ1WoFn1",
+            "lotus_16PSJLr6AKaWTgioeJbjUWFAdq7yVDG17PQ1WoFn1",
+            "lotus_16PSJMStv9sve3DfhDpiwUCa7RtqkyNBoS8RjFZSt",
+            "lotus_16PSJMStv9sve3DfhDpiwUCa7RtqkyNBoS8RjFZSt",
```
Comment (Boblepointu) [inline src/consensus/addresses.h:121]: address "...ZSt" should have 4 seat.
@schancel See telegram for explanation.

**Review by Boblepointu — CHANGES_REQUESTED** [review]

File: src/consensus/addresses.h (hunk 1 above)
Comment (schancel) [reply src/consensus/addresses.h:121]: I'm sorry but we're not doing that. That website has to be over-provisioned if it's costing that much. We will do two.

**Pull request merged**